addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -3,257 +3,234 @@
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc4 g-col-lg-3 g-height-2 col-lg-3 justify-content-center text-center" data-name="Block"
+            style="grid-area: 1 / 4 / 4 / 7; z-index: 2; background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-1) 100%);">
+            <h3 class="h4-fs">Community Focus</h3>
+            <p>Building connections</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-2 text-center o_cc o_cc2" data-name="Block"
+        <div class="o_grid_item o_cc o_cc2 g-col-lg-3 g-height-2 col-lg-3 justify-content-center text-center" data-name="Block"
             style="grid-area: 1 / 7 / 4 / 10; z-index: 3;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+            <h3 class="h4-fs">Key Milestone</h3>
+            <p>Reaching new heights together</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-2 text-center o_cc o_cc3" data-name="Block"
-            style="grid-area: 1 / 10 / 4 / 13; z-index: 4;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc3 g-col-lg-3 g-height-2 col-lg-3 justify-content-center text-center" data-name="Block"
+            style="grid-area: 1 / 10 / 4 / 13; z-index: 4; background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-2) 100%);">
+            <h3 class="h4-fs">Greater Impact</h3>
+            <p>Making a difference every day</p>
         </div>
     </div>
 </template>
 
 <template id="s_masonry_block_alternation_text_image_template" groups="base.group_user">
-    <div class="row o_grid_mode" data-row-count="10">
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-10 text-center o_cc o_cc2" data-name="Block"
-            style="grid-area: 1 / 1 / 11 / 4; z-index: 1;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+    <div class="row o_grid_mode" data-row-count="4" style="gap: 16px;">
+        <div class="o_grid_item o_cc o_cc2 g-col-lg-3 g-height-4 col-lg-3 justify-content-start rounded" data-name="Block" style="z-index: 1; grid-area: 1 / 1 / 5 / 4; --grid-item-padding-y: 20px;">
+            <h3 class="h4-fs">Increadible <br class="d-none d-lg-inline"/>Features</h3>
+            <p>&#160;</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-10 o_grid_item_image text-center" data-name="Block"
-            style="grid-area: 1 / 4 / 11 / 7; z-index: 2;">
-            <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto" alt=""/>
+        <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-4 col-lg-3 rounded" data-name="Block" style="z-index: 2; grid-area: 1 / 4 / 5 / 7;">
+            <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto rounded" alt=""/>
```

Comment:
Resolving this since, as discussed in [before](https://github.com/odoo/odoo/pull/173870#discussion_r1698185641), we'll make use of `round` and `round-[x]` classes to manage roundeness at "theme level"